azext_edge/edge/params.py

```diff
@@ -302,6 +310,822 @@ def load_iotops_arguments(self, _):
+            arg_group="Transport Layer Security(TLS)",
+        )
+        context.argument(
+            "copy_broker_props_disabled",
+            options_list=["--copy-broker-props-disabled", "--cpbd"],
+            arg_type=get_three_state_flag(),
+            help="Disable copy broker properties.",
+        )
+        context.argument(
+            "group_id",
+            options_list=["--group-id", "--gid"],
+            help="ID of consumer group that the data flow uses to read messages "
+            "from the Kafka topic."
+        )
+        context.argument(
+            "partition_strategy",
+            options_list=["--partition-strategy", "--ps"],
+            arg_type=get_enum_type(
+                KafkaPartitionStrategyType,
+                default=KafkaPartitionStrategyType.DEFAULT.value
+            ),
+            help="The partition handling strategy controls how messages are "
+            "assigned to Kafka partitions when sending them to Kafka topics.",
+        )
+        context.argument(
+            "sasl_type",
+            options_list=["--sasl-type"],
+            arg_type=get_enum_type(AuthenticationSaslType, default=None),
+            help="The type of SASL authentication.",
+            arg_group="SASL Authentication",
+        )
+        context.argument(
+            "tls_disabled",
+            options_list=["--tls-disabled"],
+            arg_type=get_three_state_flag(),
+            help="The data flow uses an insecure connection to the Kafka/MQTT broker.",
+            arg_group="Transport Layer Security(TLS)",
+        )
+        context.argument(
+            "client_id_prefix",
+            options_list=["--client-id-pre"],
+            help="The client id prefix for MQTT client. "
+            "Note: Changing the client ID prefix after IoT "
+            "Operations deployment might result in data loss.",
+        )
+        context.argument(
+            "keep_alive",
+            options_list=["--keep-alive", "--ka"],
+            help="The maximum time in seconds that the data flow client can "
+            "be idle before sending a PINGREQ message to the broker. "
+            "Min value: 0.",
+            type=int,
+        )
+        context.argument(
+            "max_inflight_messages",
+            options_list=["--max-inflight-msg", "--mim"],
+            help="the maximum number of inflight messages that the data "
+            "flow MQTT client can have. Min value: 0.",
+            type=int,
+        )
+        context.argument(
+            "protocol",
+            options_list=["--protocol"],
+            arg_type=get_enum_type(ListenerProtocol, default=ListenerProtocol.MQTT.value),
+            help="Enable or disable websockets.",
+        )
+        context.argument(
+            "qos",
+            options_list=["--qos"],
+            help="Quality of Service (QoS) level for the MQTT messages.",
```

Comment:
nit: should we include higher numbers mean x? this can also be only added for the aio mqtt endpoint type